ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1148,12 +1154,22 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
 
     private var isUsingCustomTextColors: Bool = false
 
+    private var isUsingAttributedTitle: Bool = false
```

Comment:
I think it's better to keep a nullable property of the NSAttributedString for each of the labels and compare against nil before assigning it to the label. This way you don't have to keep this boolean in sync with the state of the labels.